operator/v1alpha1/0000_20_cluster_csi_driver.crd.yaml-merge-patch

```diff
@@ -0,0 +1,12 @@
+spec:
+  validation:
+    openAPIV3Schema:
+      properties:
+        metadata:
+          properties:
+            name:
+              type: string
+              enum:
+              - aws-ebs
+              - manila
+              - ovirt
```

Comment:
Thinking again about this, would be full CSI driver name better?
```
ebs.csi.aws.com
manila.csi.openstack.org
csi.ovirt.org
```

Reply:
may be. I was thinking how we want this to look like when user does:

```
~> oc get clustercsidrievr
ebs.csi.aws.com
manila.csi.openstack.org
csi.ovirt.org

or 
~> oc get clustercsidriver
aws-ebs
```

I thought shorter names are easier to remember and edit etc because ultimately these are user facing objects.

Reply:
discussed this offline. I have changed this to full driver name.